README.md

```diff
@@ -1,27 +1,123 @@
 
-provider.tf --> Configure Google Cloud provider
+   To set the project property in the core section, run:
+```
+        gcloud config set project YOUR_PROJECT_NAME
+```
 
-terraform.tfvars --> Defining variables
+    To set the zone property in the compute section, run:
+```
+        gcloud config set compute/zone us-central1-a
+```
 
-variables-auth.tf --> Application and authentication variables
+#Setup up project name and zone as environment variable as well
 
-vm-output.tf --> Output of VM
+```
+export PROJECT_ID="YOUR_PROJECT_NAME"
+export ZONE="us-central1-a"
+```
 
-vm.tf --> Create a Ubuntu VM
+#2. Run below command in the cloud shell which will create a GCP vm
 
-# Notes
+```
+gcloud beta compute --project=$PROJECT_ID instances create deephaven --zone=us-central1-a --machine-type=n2-standard-4 --subnet=default --network-tier=PREMIUM --maintenance-policy=MIGRATE --service-account=1088976619727-compute@developer.gserviceaccount.com --scopes=https://www.googleapis.com/auth/devstorage.read_only,https://www.googleapis.com/auth/logging.write,https://www.googleapis.com/auth/monitoring.write,https://www.googleapis.com/auth/servicecontrol,https://www.googleapis.com/auth/service.management.readonly,https://www.googleapis.com/auth/trace.append --tags=deephaven,http-server,https-server --image=centos-7-v20210701 --image-project=centos-cloud --boot-disk-size=30GB --boot-disk-type=pd-balanced --boot-disk-device-name=deephaven --no-shielded-secure-boot --shielded-vtpm --shielded-integrity-monitoring --reservation-affinity=any
+```
 
-# We have used custom image for this VM which has specific required packages installed. Once the VM was ready. We created an image using below command.
 
-gcloud compute images create deephaven    --source-disk=deeephaven-test-instance-1   --source-disk-zone=us-central1-a  --project himantej-development-test
+#3. Login to the vm which we created, use below command to login to vm
 
-Check list of Google Cloud OS images --> https://cloud.google.com/compute/docs/images
+gcloud beta compute ssh --zone "us-central1-a" "deephaven" --project "$PROJECT_ID"
 
-Create the Json file for authentication --> https://cloud.google.com/iam/docs/creating-managing-service-account-keys
+
+#4. Once you login to vm copy and paste below commands and hit enter on the server.
+This will install the required packages and then clone the  deephaven git repo and start the application.
+
+These commands will run for around 20 to 25 min. We have to wait until this is finished, docker build takes time.  
+
+
+```
+#!/bin/bash
+cd /opt
+sudo yum -y update
+sudo yum install yum-utils git wget java-1.8.0-openjdk-devel gcloud -y
+sudo export JAVA_HOME=/usr/lib/jvm/jre-1.8.0-openjdk >> /etc/bashrc
+sudo yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
+sudo yum install docker-ce docker-ce-cli containerd.io -y
+sudo yum install docker-ce-17.03.0.ce-1.el7.centos docker-ce-cli-17.03.0.ce-1.el7.centos containerd.io -y
```

Comment:
Are these redundant?

Reply:
No, I tried a couple of ways but we are getting errors with centos 7. So I had to install docker in this way. 

But this finishes within 30 sec. And again as this image creation is a one-time activity. So I don't think there will be any problem. 
I hope this answers your question.